Makefile

```diff
@@ -1,72 +1,101 @@
+default: test
+
+# All builds
+
+build: build-rust build-kotlin
```

Comment:
I find it a bit confusing that `build` does these two but not `build-swift`.  I understand why, of course, since we may not be on a Mac, but I think it's equally likely that a developer would be on a Mac and doesn't care about android and therefore doesn't have the Android SDK installed.  I'd suggest either (1) removing `build` altogether or (2) just making it an alias for `build-rust` which is the only thing we can be certain the user would have the deps for.

Reply:
And probably ditto for `test` and `lint`...

Reply:
Sounds good!